sites/svelte.dev/src/routes/repl/[id]/_components/AppControls/index.svelte

```diff
@@ -88,6 +88,10 @@
 	}
 
 	async function save() {
+		if (!$session.user) {
+			alert('Looks like you are not logged in! Log in and try saving again');
```

Comment:
softened the wording a bit

```suggestion
			alert('Please log in before saving your app');
```